test/errors/implicitModule-brief.good

```diff
@@ -0,0 +1 @@
+warning in implicitModule.chpl:4: an implicit module named 'implicitModule' is being introduced to contain this file-scope code
```

Comment:
I'd probably drop "this" from this brief form just to make it a bit briefer (and I think "this" is implied by the file location).